x-pack/plugins/ml/public/data_frame/components/aggregation_list/_aggregation_label_form.scss

```diff
@@ -0,0 +1,13 @@
+.mlAggregationLabel--button {
+  width: $euiSizeL;
+}
+
+.mlAggregationLabel--text {
+  min-width: 0;
+}
+
+.mlAggregationLabel__text {
```

Comment:
This is better done by `@include euiTextTruncate;` You may not need `mlAggregationLabel__text` at all then.

Reply:
Good catch, didn't know about `eui-textTruncate`. Fixed in 0867f89.